modules/unmanaged-operators.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture-installation.adoc
+// * updating/updating-cluster-between-minor.adoc
+
+[id="unmanaged-operators_{context}"]
+= Unmanaged Operators
+
+The _management state_ of an Operator determines whether an Operator is actively
+managing the resources for its related component in the cluster as designed. If
+an Operator is set to an unmanaged state, it does not respond to changes in
+configuration nor does it receive updates.
+
+While this can be helpful in non-production clusters or during debugging,
+Operators in an unmanaged state are unsupported and the cluster administrator
+assumes full control of the individual component configurations and upgrades.
+Depending on the method the Operator was set to unmanaged, it can also render
```

Comment:
Some missing words here. Maybe:
"Depending on the method used to set the Operator to unmanaged, ..."

Reply:
I don't know that we want to say it 'render the entier cluster unsupported'. 
I get that this can be the case if the operator is a key system operator. However, this might not be the case in the majority of situations.

Reply:
@sferich888 OK, this was basically just alluding to if the method was by using a CVO override. I'll take out "Depending on the method the Operator was set to unmanaged, it can also render the entire cluster unsupported." and just leave the warning box later for "Setting CVO overrides puts the entire cluster in an unsupported state. Reported issues must be reproduced after removing the overrides for support to proceed," unless that also needs to be tweaked.